docs/user/data.rst

```diff
@@ -35,6 +35,26 @@ JupyterLab sessions by default. As a result, you do need to be aware of dealing
 with data stored in LFS if you want to use it efficiently in your work with
 renku.
 
+Uploading Data to Renkulab and Creating Datasets with the CLI
```

Comment:
```suggestion
Uploading Data to a Renkulab session to create a Dataset with the CLI
```

Reply:
Just a suggestion that could make the goal of that paragraph even a bit clearer.